lib/helper/Puppeteer.js

```diff
@@ -248,9 +248,13 @@ class Puppeteer extends Helper {
       this.debugSection('Session', 'cleaning cookies and localStorage');
       await this.clearCookie();
     }
-    await this.executeScript('localStorage.clear();').catch((err) => {
-      if (!(err.message.indexOf("Storage is disabled inside 'data:' URLs.") > -1)) throw err;
-    });
+    const currentUrl = await this.grabCurrentUrl();
+
+    if (currentUrl.startsWith('http')) {
```

Comment:
Looks like will not work for `file`, `ftp` and other protocols.
Hope nobody uses like this )